### PR TITLE
Improved ldsUtils to handle apex errors

### PR DESCRIPTION
**Pull request by rahulmalhotra**
### What does this PR do?
Improve Apex Error Handling in LDS Utils

### What issues does this PR fix or reference?

#445

## The PR fulfills these requirements:

- [x] Code linting and formatting was performed.

### Functionality Before

image

### Functionality After

image

### CODE REVIEW

File: force-app/main/default/lwc/ldsUtils/ldsUtils.js [hunk 1]
```diff
@@ -19,7 +19,26 @@ export function reduceErrors(errors) {
...
+                            return e.message;
+                        } else if (
+                            e.pageErrors &&
+                            Array.isArray(e.pageErrors)
+                        ) {
+                            let pageErrors = [];
+                            e.pageErrors.forEach((pageError) => {
+                                pageErrors.push(pageError.message);
+                            });
+                            return pageErrors.join(' ');
```
Comment (kevinv11n) [inline force-app/main/default/lwc/ldsUtils/ldsUtils.js:34]:
```suggestion
                            return e.pageErrors.map(pageError => pageError.message).join(' ');
```

Comment (rahulmalhotra) [reply force-app/main/default/lwc/ldsUtils/ldsUtils.js:34]: This is perfect! Thanks @kevinv11n . I am updating it now.

**Comment by rahulmalhotra** [comment]
Hi @kevinv11n 

Can you check and merge it now? I have done the suggested changes. Thanks!

**Comment by kevinv11n** [comment]
@pchittum  can you please review?

**Comment by pchittum** [comment]
@rahulmalhotra thanks for this. I was off until early next week and just getting to look at this. Stay tuned.

**Review by pchittum — APPROVED** [review]

**Pull request merged**